From 6e4ecb04cc15c6252ac6ef1c92a49b6325d8189f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 29 Jul 2018 17:50:09 +0300
Subject: [PATCH] board: habana: dali: update the debug UART clock

The most recent PLL settings (PCIE DIV1) are lowering the UART and
SPI clock frequencies. This garbles the UART baudrate and thus the
console isn't working correctly.
Ajust the UART clock (and thus also the SPI clock) frequency to cope
with the new PLL divisors values.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/dali/uboot_asic.fragment b/board/habanalabs/dali/uboot_asic.fragment
index f40dd75..581f779 100644
--- a/board/habanalabs/dali/uboot_asic.fragment
+++ b/board/habanalabs/dali/uboot_asic.fragment
@@ -2,7 +2,7 @@
 CONFIG_CONARGS="serial0:115200n8"
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
-CONFIG_DEBUG_UART_CLOCK=172000000
+CONFIG_DEBUG_UART_CLOCK=107500000
 CONFIG_BOOTCOMMAND="getfit && bootm $fileaddr#conf@1"
 CONFIG_DEFAULT_DEVICE_TREE="dali_asic"
 CONFIG_SPI_FLASH_MACRONIX=y
